js/webserver.js

```diff
@@ -142,18 +143,18 @@ WebServer.prototype = {
   },
 
   startListening: Promise.coroutine(function* (app) {
-    let t = this;
     if (this.config.https && this.config.https.port) {
-      let makeHttpsServer = function*(ipAddress, port) {
+      const port = this.config.https.port;
+      for (let ipAddress of this.config.http.ipAddresses) {
```

Comment:
Because we prepsocessed the config, this is guaranteed to contain at least `["0.0.0.0" ]` so no tedious checks are required and thus the code can be simplified